src/_P003_Pulse.ino

```diff
@@ -397,16 +401,16 @@ boolean Plugin_003(byte function, struct EventStruct *event, String& string)
           String par1 = parseString(string, 2); // Reset command ?
 
           int idx = 1;         
-          if (par1 == "r" || par1 == "ri" ) idx = 2;
+          if (par1 == F("r") || par1 == F("ri") ) idx = 2;
           // Allow for an optional taskIndex parameter. When not given it will take the first task of this plugin.
           if (!pluginOptionalTaskIndexArgumentMatch(event->TaskIndex, string, idx)) {
             break;
           }
 
           doStatisticLogging(event->TaskIndex, F("P003+"), P003_PULSE_STATS_ADHOC_LOG_LEVEL);
           doTimingLogging(event->TaskIndex, F("P003+"), P003_PULSE_STATS_ADHOC_LOG_LEVEL);
-          if (par1 == "i" || par1 == "ri") P003_StatsLogLevel[event->TaskIndex] = LOG_LEVEL_INFO;
-          if (par1 == "r" || par1 == "ri") resetStatsErrorVars(event->TaskIndex);
+          if (par1 == F("i") || par1 == F("ri")) P003_StatsLogLevel[event->TaskIndex] = LOG_LEVEL_INFO;
+          if (par1 == F("r") || par1 == F("ri")) resetStatsErrorVars(event->TaskIndex);
```

Comment:
You still have the same `if (...)` twice, so why not wrap it in a scope?
```c++
          if (par1 == F("i") || par1 == F("ri")) {
            P003_StatsLogLevel[event->TaskIndex] = LOG_LEVEL_INFO;
            resetStatsErrorVars(event->TaskIndex);
          }
```

Reply:
If is actually 3 conditions: ``if (par1 == F("i") || par1 == F("r") || par1 == F("ri")) {``

Reply:
> If is actually 3 conditions: `if (par1 == F("i") || par1 == F("r") || par1 == F("ri")) {` so maybe they shouldn't be combined.

Ah yep, my mistake...